integ-test/src/test/java/org/opensearch/sql/legacy/OpenSearchSQLRestTestCase.java

```diff
@@ -195,7 +195,9 @@ protected static void wipeAllOpenSearchIndices(RestClient client) throws IOExcep
         try {
           // System index, mostly named .opensearch-xxx or .opendistro-xxx, are not allowed to
           // delete
-          if (!indexName.startsWith(".opensearch") && !indexName.startsWith(".opendistro")) {
+          if (!indexName.startsWith(".opensearch")
```

Comment:
This is working till now because all the datasource related ITs are overriding the after method with out super call. WIth DataSourceEnabledIT.java, we are running the default after method which tries to wipe out all indices including system index `.ql-datasources`. Since system indices are not allowed to delete when security plugin is enabled. This is causing issues with build integration tests.